doc/contributing/pull-requests.md

```diff
@@ -525,6 +525,13 @@ to fail on specific platforms or for so-called "flaky" tests to fail ("be red").
 It is vital to visually inspect the results of all failed ("red") tests to
 determine whether the failure was caused by the changes in the pull request.
 
+### Dependencies
+
+Ideally pull requests for dependencies should be generated by automation.
+Pay special attention to pull requests for dependencies which have not
+been automatically generated and follow the guidance in
+[Maintaining Dependencies](https://github.com/nodejs/node/blob/main/doc/contributing/maintaining/maintaining-dependencies.md#updating-dependencies).
+
```

Comment:
This doesn't really belong here. If the emphasis is on "pay special attention", then this should be in the [Collaborator guide](https://github.com/nodejs/node/blob/main/doc/contributing/collaborator-guide.md).

This file (`doc/contributing/pull-requests.md`) already has an existing section on Dependencies which directs readers to `doc/contributing/maintaining/maintaining-dependencies.md`: https://github.com/nodejs/node/blob/67c19c2900506cd4c60016765e0b7954409dcf90/doc/contributing/pull-requests.md?plain=1#L34-L45

Reply:
@richardlau thanks, I'll take a look this part to the collaborator guide